refinery/core/models.py

```diff
@@ -364,10 +367,36 @@ class SharableResource(OwnableResource):
     def __unicode__(self):
         return self.name
 
+    def get_owner(self):
+        owner = None
+
+        content_type_id = ContentType.objects.get_for_model(self).id
+        permission_id = Permission.objects.filter(
+            codename='share_%s' % self._meta.verbose_name
+        )[0].id
+
+        perms = UserObjectPermission.objects.filter(
+            content_type_id=content_type_id,
+            permission_id=permission_id,
+            object_pk=self.id
+        )
+
+        if perms.count() > 0:
+            try:
+                owner = User.objects.get(id=perms[0].user_id)
+            except User.DoesNotExist:
+                pass
+
+        return owner
+
```

Comment:
Feel free to ignore, since this is just a move, but:
- There is an [`exists()`](https://docs.djangoproject.com/en/1.8/ref/models/querysets/#exists), but I'd be surprised if it's much of an improvement over `count()`.
- Maybe just return in the try? (and get rid of `owner = None` and `return owner`)
- I can see that it's tested below, but I can't remember what it would mean to be ownerless, and how that would be useful.